civitai-downloader-v2/src/ui/export.py

```
@@ -0,0 +1,881 @@
+        
+        # Max records limit
+        if filters.max_records:
+            filtered_data = filtered_data[:filters.max_records]
+        
+        return filtered_data
+    
+    def _get_record_date(self, record: Dict[str, Any], default: datetime) -> datetime:
+        """Extract date from record."""
+        date_fields = ['created_at', 'updated_at', 'timestamp', 'date']
+        
+        for field in date_fields:
```

Ruff (0.12.2)

Import `field` from line 12 shadowed by loop variable

(F402)